.travis.yml

```diff
@@ -3,7 +3,11 @@ rvm:
 - 2.2.2
 gemfile:
 - spec/dummy/Gemfile
-install: bundle install
+install:
+- bundle install
+- cd spec/dummy && bundle install
```

Comment:
@samnang @justin808 am I missing something here? I'm getting `console` is undefined on the specs